tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16.yaml

```diff
@@ -8,10 +8,8 @@ user:
   RUN_ENVIR: community
 workflow:
   CCPP_PHYS_SUITE: FV3_GFS_v16
-  DATE_FIRST_CYCL: '20190701'
-  DATE_LAST_CYCL: '20190701'
-  CYCL_HRS:
-    - 0
+  DATE_FIRST_CYCL: '2019070100'
+  DATE_LAST_CYCL: '2019070100'
```

Comment:
All of the config files are changed in this way. Removing the CYCL_HRS setting and creating the associated start and stop times based on what is here. Occasionally, the cycle frequency (`INCR_CYCL_FREQ`) is updated to reflect the actual cycling frequency for those cases where `CYCL_HRS = [0, 12]`, for example.